modules/apps/site/site-cms-site-initializer-test/src/testIntegration/java/com/liferay/site/cms/site/initializer/internal/display/context/test/BaseSectionDisplayContextTestCase.java

```diff
@@ -113,6 +113,28 @@ public HashMap<String, Object> getBaseAdditionalProps() {
 
 				return collaboratorURL;
 			}
+		).put(
+			"objectDefinitionCssClasses",
+			HashMapBuilder.put(
+				"default", "content-icon-custom-structure"
```

Comment:
Question: that literals can be / are constants?

Reply:
<img width="2464" height="980" alt="image" src="https://github.com/user-attachments/assets/7fbc2f24-eed5-459c-a2fb-239896da2c0e" />

Reply:
I was talking about the other part 
<img width="198" height="175" alt="Captura de pantalla 2025-09-04 a las 10 05 07" src="https://github.com/user-attachments/assets/676a7112-9a52-4cf4-ae74-a409ccad0d4d" />